### ISSUE TITLE
In Category layout- > Sorting, page selection, grid/list view etc. doesnt work if deployed in Azure Web Apps

### ISSUE THREAD

**Issue by Nikhil13x** [grandnode/grandnode2#251]
Since the Azure web app uses ARR, the request reaching the code will be always http, regardless of the origin was from https.

var currentPageUrl = _httpContextAccessor.HttpContext.Request.GetDisplayUrl();  --> this line in GetViewSortSizeOptionsHandler.cs  -> will be an http based URL  . it wont be https.

Hence, we will get console error as below..
Mixed Content: The page at 'https://store.com/category?orderby=0&timestamp=1649072845451' was loaded over HTTPS, but requested an insecure XMLHttpRequest endpoint 'http://store.com/category?orderby=0&timestamp=1649078904357&viewmode=grid'. This request has been blocked; the content must be served over HTTPS.

Reference: https://stackoverflow.com/questions/38501618/is-current-request-being-made-over-ssl-with-azure-deployment

**Comment by Nikhil13x** [comment]
Thanks for the info. 
Closing the issue. Will test this and reopen if required.

**Comment by Nikhil13x** [comment]
It didn't work for the Azure web app deployed from the docker image.

**Comment by Nikhil13x** [comment]
It didn't work for the Azure web app deployed from the docker image..

**Comment by KrzysztofPajak** [comment]
I use Azure web apps (not docker image) here https://grandnode.azurewebsites.net/ and It works well

**Comment by Nikhil13x** [comment]
Yes.. its working when deployed to azure web app directly. (Even without setting the UseForwardedHeaders  to true.)
Should this issue be kept open to address the issue with docker deploy on azure web app?
